node-tests/unit/commands/exam-test.js

```diff
@@ -110,6 +110,12 @@ describe('ExamCommand', function() {
         assert.equal(called.testRunOptions.query, '_split=2&_partition=2&_weighted');
       });
     });
+
+    it('should set split env var', function() {
+      return command.run({ split: 5 }).then(function() {
+        assert.equal(process.env.EMBER_EXAM_SPLIT, '5');
```

Comment:
CodeClimate doesn't like the similarity between this test and one of the other ones above. Could you look into combining them? Since they both only test the `split` option, it makes sense to merge them and just have multiple assertions.